akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -220,4 +220,11 @@ object Sink extends SinkApply {
     new Sink(new AcknowledgeSink(bufferSize, DefaultAttributes.acknowledgeSink, shape("AcknowledgeSink"), timeout))
   }
 
+  /**
+   * Crates a `Sink` that is materialized into `Source` to connect with another flow.
+   * This sink gets backpressure from second flow and does not have internal buffer.
+   */
+  def pipedSink[T](): Sink[T, Source[T, Unit]] =
+    Sink.publisher[T].mapMaterializedValue(Source(_)).named("PipedSink")
```

Comment:
@ktoso you have no other choice, since after the materialization of this Sink the upper part of the stream will be already running. At that point there is no fusing possible.